api/src/services/monitoring.js

```diff
@@ -179,7 +252,7 @@ const getReplicationLimitLog = () => {
     });
 };
 
-const json = () => {
+const jsonV1 = () => {
```

Comment:
is this considered deprecated? If yes, can we add a deprecation message or a comment?

Or is it that jsonV1 is going to be extended in the next version (jsonV2)?

Reply:
Yea, I don't think we historically added deprecation messages (we always managed to be backwards compatible somehow), but I guess we could start. 
I'm adding a `deprecation` middleware for this.